<launch>
	<!-- Possible values: simu, draco_r, magnus -->
	<arg name="drone_real"     default="draco_r" />

	<arg name="trajectory_gen" default="false" />
	<arg name="display"        default="false" />
	<arg name="rosbag_record"  default="false" />

	<node pkg="trajectory_control" type="trajectory_control_node" name="trajectory_control_node_real" output="screen" >
		<param name="simulated_env" value="false" />
		<rosparam file="$(find trajectory_control)/config/$(arg drone_real)_params.yaml" command="load"/>
	</node>

  <node pkg="trajectory_control" type="trajectory_gen.py" name="trajectory_gen_node" if="$(arg trajectory_gen)"></node>

	<node pkg="rosbag" type="record" name="rosbag_record_cam"
		args="record -o $(find trajectory_control)/rosbag/bag mavros/local_position/odom mavros/imu/data mavros/rc/out mavros/setpoint_raw/attitude"
		if="$(arg rosbag_record)">
	</node>

	<node pkg="trajectory_control" type="display.py" name="display_node" if="$(arg display)"></node>

</launch>
